src/base/utils/foreignapps.h

```diff
@@ -47,6 +47,8 @@ namespace Utils::ForeignApps
 
         QString executableName;
         Version version;
+
+        inline static const Version MINIMUM_SUPPORTED_VERSION {3, 9, 0};
```

Comment:
Isn't `inline` redundant here?

Reply:
It is required since it is `static`.

Reply:
> It is required since it is `static`.

Ok. It is to avoid "out-of-class" definition.